config.go

```diff
@@ -29,6 +29,35 @@ import (
 	"k8s.io/klog/v2"
 )
 
+type ChainValidationConfig struct {
+	// Path to the file containing root certificates that are acceptable to the
```

Comment:
Nit on doc comments in this struct - they should really start with the name of the thing being commented on, e.g. "`RootsPEMFile` is the path to a file con..."

Probably worth a comment on the struct too since it's exported?